library/Jira/MonitoringInfo.php

```diff
@@ -164,7 +164,7 @@ public function getLastStateChange()
             return DateFormatter::formatDateTime($this->object->last_state_change);
         }
 
-        return DateFormatter::formatDateTime($this->object->state->last_state_change);
+        return DateFormatter::formatDateTime($this->object->state->last_state_change->getTimestamp());
```

Comment:
This requires Icinga DB Web in version 1.1. Though, we should not raise the minimum required version with a minor release. Please add support for both cases. (it being a `DaetTime` or `integer`)